base.gradle

```diff
@@ -145,7 +146,8 @@ android {
         debugProd {
             debuggable true
 
-            testCoverageEnabled false
+            enableUnitTestCoverage = false
+            enableAndroidTestCoverage = false
```

Comment:
This change was because of the following message:

> Migrate testCoverageEnabled to enableUnitTestCoverage and enableAndroidTestCoverage
> Starting with Android Gradle Plugin 7.3.0 testCoverageEnabled has been replaced by enableUnitTestCoverage and enableAndroidTestCoverage. It will be removed in AGP 9.0.0.